memorystore/valkey/caching/samples/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>samples</groupId>
```

Comment:
populate groupId and artifactdId similar to https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/memorystore/redis/pom.xml ?